app/views/layouts/application.slim

```diff
@@ -1,29 +1,40 @@
 doctype html
-html
+html lang="en"
   head
     meta charset="utf-8"
     meta name="ROBOTS" content="NOODP"
     title Afterwork
+    meta name="viewport" content="width=device-width, initial-scale=1.0"
```

Comment:
Add this one as well `meta http-equiv="x-ua-compatible" content="ie=edge"`
Read [this](https://github.com/h5bp/html5-boilerplate/blob/master/src/doc/html.md#x-ua-compatible)